.github/workflows/build_deploy.yml

```diff
@@ -189,20 +191,18 @@ jobs:
   upload_pypi:
     needs: [build_wheels_py27_35, build_wheels_py3, test_alpine_sdist]
     runs-on: ubuntu-latest
-    if: (github.event_name == 'release' && github.event.action == 'published') || (github.event_name == 'workflow_dispatch')
+    if: (github.event_name == 'release' && github.event.action == 'published')
     steps:
       - uses: actions/download-artifact@v3
         with:
           name: artifact
           path: dist
 
       - uses: actions/checkout@v3
-        if: github.event_name == 'workflow_dispatch'
         # Include all history and tags
         with:
           fetch-depth: 0
       - name: Validate deploy version
-        if: github.event_name == 'workflow_dispatch'
         run: |
           ./scripts/validate-version "${{ github.event.inputs.expectedVersion }}"
```

Comment:
All of these lines can go away. we don't need these steps at all if we aren't manually triggering.